public/app/features/integrated-alerting/components/AlertRules/AlertRuleParamField/AlertRuleParamField.messages.ts

```diff
@@ -0,0 +1,27 @@
+import { TemplateFloatParam, TemplateParamUnit } from '../../AlertRuleTemplate/AlertRuleTemplate.types';
+import { beautifyUnit } from '../../AlertRuleTemplate/AlertRuleTemplate.utils';
+
+export const Messages = {
+  getFloatDescription: (name: string, summary: string, unit: TemplateParamUnit, float?: TemplateFloatParam) => {
+    if (!float) {
+      return '';
+    }
+
+    const lowerCaseNameSentenceArr = name.toLowerCase().split(' ');
+    const capitalizedSentence = lowerCaseNameSentenceArr
+      .map(word => `${word[0].toUpperCase()}${word.slice(1)}`)
+      .join(' ');
+    const { has_min, has_max, min = 0, max = 0 } = float;
+    const paramDetails: string[] = [beautifyUnit(unit)];
```

Comment:
```suggestion
    const paramDetails: string[] = [
        beautifyUnit(unit),
        has_min ? `min: ${min}` : '',
        has_max ? `max: ${max}` : '',
    ];
```

This is just personal preference, but choose what you prefer please. Also if we could use camel case (e.g. `hasMin` instead of `has_min`) to keep consistency it would be good.

Reply:
The problem with that approach is that on the last line I do a `paramDetails.join(',')`, so it could happen something like `seconds,,` 😕 
The snake case there is because that field is the from the API. What do you guys usually do in these cases? Should I rename the interface from `TemplateFloatParam` to `TemplateFloatParamAPI`, so it's clear it's a direct reading?

Reply:
You could use `filter` before `join` to remove the empty spaces. But that would require an additional iteration so let's leave it as it is.
Normally I add a "middle layer" to convert whatever comes from the API to our "model", either through a function after then response from the API or through the `then` function (which I prefer because it can be done on the API layer and still return a Promise, something like [this](https://github.com/percona-platform/grafana/blob/37985de89b0ce7e465baee009d5c7c97be09f0b4/public/app/percona/dbaas/components/DBCluster/PSMDB.service.ts#L74))

Reply:
Great, thanks for the tip 👍